frontend/src/screens/wallet/Swaps.tsx

```diff
@@ -0,0 +1,499 @@
+function SwapInForm() {
+  const { toast } = useToast();
+  const { data: swapsSettings, mutate } = useSwaps();
+
+  const [swapAmount, setSwapAmount] = useState("");
+  const [destination, setDestination] = useState("");
+  const [isInternalSwap, setInternalSwap] = useState(true);
+  const [loading, setLoading] = useState(false);
+
+  const onSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+
+    try {
+      setLoading(true);
+      await request("/api/wallet/swaps", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          // TODO: review conflict between swap in/out
+          swapAmount: parseInt(swapAmount),
+          destination: destination, // TODO: assume empty destination as dest -> "Spending Balance"
+        }),
+      });
+      toast({ title: "Saved successfully." });
+      await mutate();
+    } catch (error) {
+      toast({
+        title: "Saving swap settings failed",
+        description: (error as Error).message,
+        variant: "destructive",
+      });
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  const paste = async () => {
+    const text = await navigator.clipboard.readText();
+    setDestination(text.trim());
+  };
+
+  if (!swapsSettings) {
+    return <Loading />;
+  }
+
+  return (
+    <form onSubmit={onSubmit} className="flex flex-col gap-4">
+      <h2 className="text-muted-foreground">
+        Swap on-chain funds into your lightning spending balance.
+      </h2>
+      <div className="grid gap-1.5">
+        <Label>On-chain to Lightning swap</Label>
+        <Input
+          type="number"
+          placeholder="Amount in satoshis"
+          value={swapAmount}
+          min={MIN_AUTO_SWAP_AMOUNT}
+          onChange={(e) => setSwapAmount(e.target.value)}
+          required
+        />
+        <p className="text-xs text-muted-foreground">
+          Minimum {new Intl.NumberFormat().format(MIN_AUTO_SWAP_AMOUNT)} sats
+        </p>
+      </div>
+      <Label>Swap To</Label>
+      <RadioGroup
+        defaultValue="normal"
+        value={isInternalSwap ? "internal" : "external"}
+        onValueChange={() => {
+          setDestination("");
+          setInternalSwap(!isInternalSwap);
+        }}
+        className="flex gap-4 flex-row"
+      >
+        <div className="flex items-start space-x-2 mb-2">
+          <RadioGroupItem value="internal" id="internal" className="shrink-0" />
+          <Label
+            htmlFor="internal"
+            className="text-primary font-medium cursor-pointer"
+          >
+            Spending balance
+          </Label>
+        </div>
+        <div className="flex items-start space-x-2">
+          <RadioGroupItem value="external" id="external" className="shrink-0" />
+          <Label
+            htmlFor="external"
+            className="text-primary font-medium cursor-pointer"
+          >
+            External lightning wallet
```

Comment:
![image](https://github.com/user-attachments/assets/7d9d9415-aa1f-4924-88d3-c961fb2618df)

Can we remove the external option? it makes no sense. We want people to use Alby Hub, not another lightning wallet. The reason it's there for swapping out is so you can swap to cold storage.